src/sqlfluff/core/default_config.cfg

```diff
@@ -43,6 +43,8 @@ comma_style = trailing
 allow_scalar = True
 single_table_references = consistent
 unquoted_identifiers_policy = all
+semi_colon_new_line = False
+require_final_semi_colon = False
```

Comment:
```suggestion
require_final_semicolon = False
```

Reply:
Flowed these changes through to code and unit tests